testsuite/forge-cli/src/main.rs

```diff
@@ -1346,6 +1359,47 @@ fn gather_metrics() -> ForgeConfig {
         .add_network_test(GatherMetrics)
 }
 
+fn netbench_config_100_megabytes_per_s() -> NetbenchConfig {
```

Comment:
nit: `*_per_s` reads odd to me. Why not just complete it? `*_per_sec`? Could just be me though 😵‍💫